src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -209,89 +235,51 @@ private static void RaiseFileLoadedEvent(Mock<ITextDocument> mockDocument)
             mockDocument.Raise(x => x.FileActionOccurred += null, args);
         }
 
+        private static void RaiseBufferChangedEvent(Mock<ITextBuffer> mockTextBuffer, ITextSnapshot beforeSnapshot, ITextSnapshot afterSnapshot)
+        {
+            var args = new TextContentChangedEventArgs(beforeSnapshot, afterSnapshot, EditOptions.None, null);
+            mockTextBuffer.Raise(x => x.ChangedLowPriority += null, args);
+        }
+
         private void CheckAnalysisWasNotRequested()
         {
             mockAnalyzerController.Verify(x => x.ExecuteAnalysis(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<IEnumerable<AnalysisLanguage>>(),
                 It.IsAny<IIssueConsumer>(), It.IsAny<IAnalyzerOptions>(), It.IsAny<CancellationToken>()), Times.Never);
         }
-
-        #endregion
-
-        #region Processing analysis results tests
-
-        [TestMethod]
-        public void WhenNewIssuesAreFound_ButForIncorrectFile_FilterNotCalledAndListenersAreNotUpdated()
```

Comment:
Removed two tests: now the responsibility of the accumulating issue consumer and tested there.